### PR TITLE
[Release 1.7.0] Update module versions

### PR DESCRIPTION
**Pull request by shahabhijeet**
This checklist is used to make sure that common issues in a pull request are covered by the creator. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270%28v=vs.85%29.aspx).

Below in **Overall Changes**, check off the boxes that apply to your PR. Within each of the categories that you select, make sure that you can check off **all** of the boxes.
## Overall Changes
- [x] [**MANDATORY** - General changes](#general)
- [x] [Add/remove/edit cmdlet(s)](#cmdlet-signature)
### General
- [x] Title of the PR is clear and informative
- [x] There are a small number of commits that each have an informative message
- [x] If it applies, references the bug/issue that the PR fixes
- [x] All files have the Microsoft copyright header
- [x] Cmdlets refer to management libraries through nuget references - no dlls are checked in
- [ ] The PR does not introduce breaking changes (unless a major version change occurs in the assembly and module)

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@shahabhijeet__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (abhishah). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/CognitiveServices/AzureRM.CognitiveServices.psd1 [hunk 1]
```diff
@@ -48,7 +48,7 @@ CLRVersion='4.0'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @( @{ ModuleName = 'AzureRM.Profile'; ModuleVersion = '1.0.10'})
+RequiredModules = @( @{ ModuleName = 'AzureRM.Profile'; ModuleVersion = '1.0.11'})
```
Comment (markcowl) [inline src/ResourceManager/CognitiveServices/AzureRM.CognitiveServices.psd1]: ??

File: src/ServiceManagement/Services/Commands.Utilities/Azure.psd1 [hunk 2]
```diff
@@ -9,7 +9,7 @@
 @{
 
 # Version number of this module.
-ModuleVersion = '1.6.0'
+ModuleVersion = '1.7.0'
```
Comment (markcowl) [inline src/ServiceManagement/Services/Commands.Utilities/Azure.psd1]: 1.6.1

**Comment by markcowl** [comment]
signing job here:  http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-sign/584/

File: src/ResourceManager/MachineLearning/AzureRM.MachineLearning.psd1 [hunk 3]
```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '0.9.2' 
+ModuleVersion = '0.9.3'
```
Comment (markcowl) [inline src/ResourceManager/MachineLearning/AzureRM.MachineLearning.psd1]: We already released version 0.9.3.  We should make this 0.9.2.1

File: src/ResourceManager/AzureStackAdmin/AzureRM.AzureStackAdmin.psd1 [hunk 4]
```diff
@@ -12,7 +12,7 @@
 ModuleToProcess = 'Microsoft.AzureStack.Commands.dll'
 
 # Version number of this module.
-ModuleVersion = '0.9.8'
+ModuleVersion = '0.9.9'
```
Comment (markcowl) [inline src/ResourceManager/AzureStackAdmin/AzureRM.AzureStackAdmin.psd1]: 0.9.8.1

File: src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1 [hunk 5]
```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '0.9.9' 
+ModuleVersion = '0.9.10'
```
Comment (markcowl) [inline src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1]: 0.9.9.1

File: tools/AzureRM/AzureRM.psd1 [hunk 6]
```diff
@@ -49,41 +49,41 @@ ProcessorArchitecture = 'None'
...
+    @{ ModuleName = 'AzureRM.DataFactories'; RequiredVersion = '1.0.12'},
+    @{ ModuleName = 'AzureRM.DataLakeAnalytics'; RequiredVersion = '1.1.4'},
+    @{ ModuleName = 'AzureRM.DataLakeStore'; RequiredVersion = '1.0.12'},
+    @{ ModuleName = 'AzureRM.DevTestLabs'; RequiredVersion = '1.0.3'},
+    @{ ModuleName = 'AzureRM.Dns'; RequiredVersion = '1.0.12'},
+    @{ ModuleName = 'AzureRM.HDInsight'; RequiredVersion = '1.1.4'},
+    @{ ModuleName = 'AzureRM.Insights'; RequiredVersion = '1.0.12'},
+    @{ ModuleName = 'AzureRM.KeyVault'; RequiredVersion = '1.1.11'},
+    @{ ModuleName = 'AzureRM.LogicApp'; RequiredVersion = '1.0.8'},
+    @{ ModuleName = 'AzureRM.MachineLearning'; RequiredVersion = '0.9.3'},
```
Comment (markcowl) [inline tools/AzureRM/AzureRM.psd1]: when you make by hand changes to versions, make sure these versions match

File: src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin/Properties/AssemblyInfo.cs [hunk 7]
```diff
@@ -29,5 +29,5 @@
 
 [assembly: Guid("0B02390C-8AA9-4D99-8AA8-2A9D2D39682F")]
 
-[assembly: AssemblyVersion("0.9.8")]
-[assembly: AssemblyFileVersion("0.9.8")]
+[assembly: AssemblyVersion("0.9.9")]
```
Comment (cormacpayne) [inline src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin/Properties/AssemblyInfo.cs]: 0.9.8.1

File: src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage.Tests/Properties/AssemblyInfo.cs [hunk 8]
```diff
@@ -32,6 +32,6 @@
 
 [assembly: Guid("5802D147-3EE9-4352-A0B1-1C16B6DBCA7C")]
 
-[assembly: AssemblyVersion("0.9.9")]
-[assembly: AssemblyFileVersion("0.9.9")]
+[assembly: AssemblyVersion("0.9.10")]
```
Comment (cormacpayne) [inline src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage.Tests/Properties/AssemblyInfo.cs]: 0.9.9.1

File: src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Properties/AssemblyInfo.cs [hunk 9]
```diff
@@ -29,5 +29,5 @@
 
 [assembly: Guid("D4CA0CC1-CD0A-4CE2-A40D-2D8A082D8791")]
 
-[assembly: AssemblyVersion("0.9.9")]
-[assembly: AssemblyFileVersion("0.9.9")]
+[assembly: AssemblyVersion("0.9.10")]
```
Comment (cormacpayne) [inline src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Properties/AssemblyInfo.cs]: 0.9.9.1

File: src/ResourceManager/MachineLearning/Commands.MachineLearning.Test/Properties/AssemblyInfo.cs [hunk 10]
```diff
@@ -33,6 +33,6 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 
-[assembly: AssemblyVersion("0.9.2")]
-[assembly: AssemblyFileVersion("0.9.2")]
+[assembly: AssemblyVersion("0.9.3")]
```
Comment (cormacpayne) [inline src/ResourceManager/MachineLearning/Commands.MachineLearning.Test/Properties/AssemblyInfo.cs]: 0.9.2.1

File: src/ResourceManager/MachineLearning/Commands.MachineLearning/Properties/AssemblyInfo.cs [hunk 11]
```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.9.2")]
-[assembly: AssemblyFileVersion("0.9.2")]
+[assembly: AssemblyVersion("0.9.3")]
```
Comment (cormacpayne) [inline src/ResourceManager/MachineLearning/Commands.MachineLearning/Properties/AssemblyInfo.cs]: 0.9.2.1

File: tools/AzureRM/AzureRM.psd1 (hunk 6 above)
Comment (karravi-msft) [inline tools/AzureRM/AzureRM.psd1]: AzureRM.Dns version should be 2.0.1 and not 1.0.12

**Comment by cormacpayne** [comment]
Signed job: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-sign/585/

**Pull request merged**